lib/dbservice_web/controllers/user_controller.ex

```diff
@@ -54,8 +54,8 @@ defmodule DbserviceWeb.UserController do
     response(201, "Created", Schema.ref(:User))
   end
 
-  def create(conn, params) do
-    with {:ok, %User{} = user} <- Users.create_user(params) do
+  def create(conn, %{"user" => user_params}) do
```

Comment:
Why are we sending it as a struct?

We never send the parameters as a struct while making any post request.

Revert this!

Reply:
removed